client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/InvestigationTableFooter.tsx

```diff
@@ -58,6 +68,15 @@ const InvestigationTableFooter: React.FC<Props> = React.forwardRef((props: Props
         }
     ]
 
+    useEffect(() => {
+        let investigationsToGroup : InvestigationTableRow[] = [];
+        checkedRowsIds.forEach((checkedRowId: number) => {
```

Comment:
Cant you just filter tableRows to rows that are included on checkedRowsIds? If not, at least change investigationsToGroup to const

Reply:
maybe a useMemo and not a state...